services/core-web/common/reducers/partiesReducer.js

```diff
@@ -63,6 +63,11 @@ export const partiesReducer = (state = initialState, action) => {
         ...state,
         inspectors: action.payload.records,
       };
+    case actionTypes.STORE_PROJECT_LEADS:
+      return {
+        ...state,
+        project_leads: action.payload.records,
+      };
```

Comment:
This should be camelCase - 'projectLeads', and it should be initiated in the initialState as well